### PR TITLE
Create and add pull request template (#73 and #72)

### PR DESCRIPTION
**Pull request by strawberrywz**
I made a pull request template and modified the pull_request_template.md

### CODE REVIEW

**Comment by nang-dev** [comment]
I love it thank u sm strawberry

File: .github/pull_request_template.md [hunk 1]
```diff
@@ -1,7 +1,21 @@
...
+
+## Contributor License Agreement (CLA)
+Before we can accept your pull request, you'll need to sign a [Contributor License Agreement (CLA)](Contributor-License-Agreement). It is an automated process and you only need to do it once.
+
+## Description
+Please include a summary of the change and which issue is fixed. Make sure to link the issue in the pull request.
+
+Fixes # (issue)
+
+## Important Notes
```
Comment (Fryingpannn) [inline .github/pull_request_template.md:11]: I think can remove the "Important Notes", no need to mention to not edit structure of repo

File: .github/pull_request_template.md [hunk 2]
```diff
@@ -1,7 +1,21 @@
...
+Please do not edit the structure of the repo. Pear is constantly changing, and we want to make sure that we can easily integrate your changes into our codebase.
+
+You can also directly ping the maintainers or admins in the [Discord](https://discord.gg/7QMraJUsQt).
+
+## Checklist
+* [ ] I have signed the [Contributor License Agreement (CLA)](Contributor-License-Agreement)
```
Comment (Fryingpannn) [inline .github/pull_request_template.md:17]: Can remove checklist for now. Things aren't standardized yet so we don't necessarily have a checklist to follow.

File: .github/pull_request_template.md [hunk 3]
```diff
@@ -1,7 +1,21 @@
...
-* Ensure that the code is up-to-date with the `main` branch.
-* Include a description of the proposed changes and how to test them.
--->
+# Thank you for submitting a Pull Request
+
+## Contributor License Agreement (CLA)
+Before we can accept your pull request, you'll need to sign a [Contributor License Agreement (CLA)](Contributor-License-Agreement). It is an automated process and you only need to do it once.
+
+## Description
+Please include a summary of the change and which issue is fixed. Make sure to link the issue in the pull request.
```
Comment (Fryingpannn) [inline .github/pull_request_template.md:7]: This is the core part, let's also add to post screenshot or demo if needed.

Comment (strawberrywz) [reply .github/pull_request_template.md:7]: I made some changed and commited.

**Comment by nang-dev** [comment]
@strawberrywz do you mind reopening? this was supposed to be merged

**Pull request closed without merging**